Tools/TestWebKitAPI/Tests/WTF/cocoa/TypeCastsCocoa.mm

```diff
@@ -154,7 +154,8 @@ static size_t helloWorldCStringLength()
         AUTORELEASEPOOL_FOR_ARC_DEBUG {
             objectNS = adoptNS([[NSString alloc] initWithFormat:@"%s", helloWorldCString]);
             objectNSPtr = reinterpret_cast<uintptr_t>(objectNS.get());
-            EXPECT_EQ(objectNS.get(), checked_objc_cast<NSObject>(objectNS.get()));
+            NSObject* objPtr = objectNS.get();
+            EXPECT_EQ(objectNS.get(), objPtr);
```

Comment:
By removing the call to `checked_objc_cast<>()`, this isn't testing that function any longer (in a test function about `checked_objc_cast`).

Are we going to create an "upcast" for Objective-C objects?  Or will that operation be unchecked?

Reply:
No, we're not gonna create an `upcast` since such a cast operation is always implicit & safe.